drafts/WEP0001.rst

```diff
@@ -0,0 +1,273 @@
+Other than typical details like their name (both legal and preferred),
+and contact details, the form should ask roughly these questions:
+
+* What they do with documentation
+* Why they want to attend Write the Docs
+* An estimate for their costs (provide tips for looking on booking, airbnb,
+  google flights and possibly others), split out by cost item.
+  Travel and accommodation are typical costs, but applicants may specify
+  and ask for any costs they need covered to be able to attend.
+* Which country they live in
+* Whether they are part of an underrepresented or marginalised group in tech
+
+For the last question, we will clarify that underrepresented or marginalised 
+groups in tech include, but are not limited to:
+
+* women and other gender minorities of all expressions and identities;
+  e.g. trans, agender and non-binary people
+* people of color
+* sexuality minorities, including asexual people
+* people with disabilities, both visible and invisible
+* neurodivergent people
+* people with chronic illnesses or diseases
+* religious and ethnic minorities
+* age minorities (under ~21, over ~50)
+* people experiencing poverty
+* homeless and home/food insecure people
+* caregivers of children or other dependents
+* people who have experienced trauma and its aftermath (PTSD, anxiety, etc)
+* people living with or recovering from substance abuse
+
+Notably, applicants are never asked to identify which group they belong to.
+
+Selection process
+-----------------
+
+All applications will be scored by the grant review team, which should
+consist of around four reviewers. The suggested tool is Grorg, but the
+grant review team may choose different tools.
+
+Reviewers can go through all applications one by one, and score them,
+possibly with a comment. Scores, applicant names, and comments from others
+should not be shown until the reviewer has provided a score. Scores can be
+updated, but these updates will be logged and visible.
+
+The scoring should be based on the overall impact that granting this person
+will have on them, the Write the Docs community, and the applicants wider
+community and country, compared to others.
+This is difficult to assess from a short form. Several guidelines are:
+
+* To what extent will letting this person attend have a further impact on the
+  community? For example, someone who essentially wrote “I will help my
+  company make more money” should likely be scored lower than someone who is
+  active in their local community.
+* The social impact of their work can be a part too. “I work to help this
+  bank I work for become richer” has less social impact than supporting someone
+  who said “I work on projects to help addicted homeless people towards recovery”,
+  and therefore the latter should be scored higher.
+* Where is this person from? If they would take a large budget, but live in an
+  area that has many other conferences closer by, they may not be a great choice.
```

Comment:
This almost captures it, but I think there's a bit more here. Maybe we need a line item like:

* Is there another WTD conference closer to the attendee? We generally try to send folks to the closest and cheapest option in our series of conferences. Someone in Germany for example should apply to Prague instead of Portland.

Reply:
Agreed, this has been clarified.